src/deprecated.jl

```diff
@@ -2,8 +2,11 @@
 # Deprecate RodriguesVec => RotationVec
 Base.@deprecate_binding RodriguesVec RotationVec true
 
-# Deprecate Quat => UnitQuaternion
-Base.@deprecate_binding Quat UnitQuaternion true
+# Deprecate Quat => QuatRotation
+Base.@deprecate_binding Quat QuatRotation true
```

Comment:
You can probably remove this old deprecation.

Reply:
The old deprecations for `RodriguesVec`, `SPQuat` and `Quat` has been introduced in this commit (#116):
https://github.com/JuliaGeometry/Rotations.jl/pull/116/commits/793d384d14e688887b02914e2cad9fdfbdfbdb65#diff-0c3a3096ea8aa5548f7facdd7eaa649293a64f785376cca25e23f14d6af4301d

If we can remove the deprecation for `Quat`, then I think the others also can be removed.
The old type names were already deprecated before `v1.0.0` release, so I thought it's okay to remove them.

Reply:
Sure, sounds fine.